bedrock/firefox/tests/test_views.py

```diff
@@ -508,19 +508,25 @@ def test_yandex_scene_1(self, render_mock):
         req = RequestFactory().get('/firefox/new/')
         req.locale = 'ru'
         view = views.NewView.as_view()
-        view(req)
+        response = view(req)
         template = render_mock.call_args[0][1]
-        assert template == ['firefox/new/trailhead/download-yandex.html']
+        assert template == ['firefox/new/desktop/download.html']
+        doc = pq(response.content)
+        robots = doc('main[class*="t-yandex"]')
```

Comment:
@pmac The switch used to control the template but now it controls an include on the page. I'd like to update the tests to look at the response but this test is failing with a `lxml.etree.ParserError: Document is empty` Can you spot my mistake?

Reply:
I think the issue is that you're mocking the render function, which means it will not be returning a rendered page, but rather a mock object.